app/models/participant_declaration/npq.rb

```diff
@@ -70,4 +70,8 @@ def qualification_type
 
     nil
   end
+
+  def latest_outcome_of_declaration
+    outcomes.order(completion_date: :desc)&.first
```

Comment:
How about `outcomes.order(completion_date: :desc).limit(1)`?

Reply:
Yea that would also work as the same in this case

Reply:
I would still prefer `limit(1)` as it lower risk, but in this case we wouldn't have many outcomes, for `first` is fine.

Reply:
Switched to limit and resolving this comment